qiita_pet/templates/study_base.html

```diff
@@ -3,6 +3,7 @@
 <link rel="stylesheet" src="/static/vendor/css/vis.css" type="text/css">
 <script type="text/javascript" src="/static/vendor/js/vis.js"></script>
 <script type="text/javascript">
+  var curr_prep = -1;
```

Comment:
If you are using global variables, please add a comment on what are they doing and why they're needed. Easier to maintain the code